iomirea_rpc/server.py

```diff
@@ -116,14 +130,14 @@ def remove_command(self, index: int) -> _CommandType:
         await self._handler(channels[0])
 
     def run(self, *args: Any, **kwargs: Any) -> None:
-        """A blocking way to start server. Takes same arguments as Server.start."""
+        """A blocking way to start server. Takes same arguments as Client.start."""
```

Comment:
I think it is `Server`

Reply:
Woops seems I messed up there

Reply:
Actually, no. It's in the server.py file